daisy_workflows/image_import/import_image.sh

```diff
@@ -235,7 +235,7 @@ ensureCapacityOfDisk "${DISKNAME}" "${SIZE_GB}" "${ZONE}" /dev/sdc
 
 # Convert the image and write it to the disk referenced by $DISKNAME.
 # /dev/sdc is used since it's the third disk that's attached in inflate_file.wf.json.
-if ! out=$(qemu-img convert "${IMAGE_PATH}" -p -O raw -S 512b /dev/sdc 2>&1); then
```

Comment:
Why is `-S 512b` removed?

Reply:
Good find -- to keep this PR focused, I added it back, and updated the PR's overview to show the times with it included.

For background, Migrate for Compute removed that in their version -- they thought it was a no-op since the target is a block device, not a file, and that the flag was intended for creating [sparse files](https://en.wikipedia.org/wiki/Sparse_file).

Reply:
Yeah, sounds it might be a no-op, but we should understand why it was added in the first place before removing it.